.github/workflows/greet_messege.yml

```diff
@@ -19,17 +16,7 @@ jobs:
             Thanks for opening an issue :blue_heart: your issue will be seen by maintainers soon.\n
             Contributors :people_holding_hands:  like you make the open source community :earth_africa:  such an amazing place to learn :book: ,
              inspire :angel:, and create :art: .\nWe will investigate :eyes:  and get back to you as soon as possible :+1: . 
-             Just make sure you have given us sufficient information :information_source:.\n\nBy that time enjoy this joke :point_down: , 
+             Just make sure you have given us sufficient information :information_source:. And followed the [contributing](https://github.com/sButtons/sbuttons/blob/master/CONTRIBUTING.md) guidelines.\n\nBy that time enjoy this joke :point_down: , 
```

Comment:
As it was mentioned [here](https://github.com/sButtons/sbuttons/pull/872#issuecomment-714204854) please remove all new lines

Reply:
I think now it's ok sorry for delay.